lib/db_utils.pm

```diff
@@ -202,42 +194,41 @@ sub check_postgres_db {
     my $image = shift;
     my $db_ip = get_var('POSTGRES_IP');
     my $db_port = get_var('POSTGRES_PORT', '5444');
+    my $db_db = get_var('POSTGRES_DB', 'size');
     my $db_log = "/tmp/db.log";
-    # Check if the running job can push data to the Data Base
-    # We only allow this on certain conditions:
-    #  - The job must contain IMAGE_MON_SERVER
-    #  - The job shouldn't be a verification run
-    #  - The job must be executed from OSD or O3
-    return 0 if (!$db_port && !$db_ip);
-
-    # CASEDIR var is always set, and if when no specified, the value is "sle" for OSD and "opensuse" for O3
+
+    ## We only allow data push to the database if all of the following conditions are met:
+    ## 1. The job must hold the POSTGRES_IP setting (database host)
+    ## 2. The job shouldn't be a verification run
+    ## 3. The job must be executed from OSD or O3
+
+    ## Check if database host ist set
+    return 0 unless ($db_ip);
+
+    ## Check if job is a verification run
+    # CASEDIR var is always set (check vars.json). If not explicitly specified, the value is "sle/sle-micro" for OSD and "opensuse/leap-micro" for O3
     return 0 if (get_required_var('CASEDIR') !~ m/^sle$|^opensuse$|^(sle|leap)-micro$/);
 
+    ## Check if job is executed on OSD/O3
     my $job_url;
     my $openqa_host = get_required_var('OPENQA_HOSTNAME');
     if ($openqa_host =~ /openqa1-opensuse|openqa.opensuse.org/) {    # O3 hostname
         $job_url = 'https://openqa.opensuse.org/tests/' . get_current_job_id();
-    }
-    elsif ($openqa_host =~ /openqa.suse.de/) {    # OSD hostname
+    } elsif ($openqa_host =~ /openqa.suse.de/) {    # OSD hostname
```

Comment:
Let's strip the scheme adding `$openqa_host =~ s/^https?:\/\///;` above and using eq instead:

```suggestion
    } elsif ($openqa_host eq = "openqa.suse.de" {    # OSD hostname
```

Reply:
I rather would not do this, as I don't know if that will work on all workers. I'm not sure how the hostname propagates there, and we know that this already works.

So except for "it look better that way" is there a functional reason why this suggestion should be included in this PR?